content/scout/policy/remediation.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Remediation with Docker Scout
+description: Learn how Docker Scout can help you improve your software quality automatically, using remediation
+keywords: scout, supply chain, security, remediation, automation
+---
+
+Docker Scout helps you remediate supply chain or security issues by providing
+recommendations based on policy evaluation results. Recommendations are
+suggested actions you can take that improve policy compliance, or that add
+metadata to images which enables Docker Scout to provide better evaluation
+results and recommendations.
+
+Docker Scout provides remediation advice for the following policies:
+
+- [Outdated base images](#outdated-base-image-remediation)
+- [Supply chain attestations](#supply-chain-attestations-remediation)
+
+For images that violate policy, the recommendations focus on addressing
+compliance issues and fixing violations. For images where Docker Scout is
+unable to determine compliance, recommendations show you how to meet the
+prerequisites that ensure Docker Scout can successfully evaluate the policy.
+
+## View recommendations
+
+Recommendations appear on the policy details pages of the Docker Scout
+Dashboard. To get to this page:
+
+1. Open the [Docker Scout Dashboard](https://scout.docker.com/)
+2. Go to the **Policies** page.
+3. Select a policy in the list.
+
+The policy details page groups evaluation results into two different tabs
+depending on the policy status:
+
+- Violations
+- Compliance unknown
+
+The **Violations** tab lists images that don't comply with the selected policy.
+The **Compliance unknown** tab lists images that Docker Scout is unable to
+determine the compliance status for. When compliance is unknown, Docker Scout
+needs more information about the image.
+
+To view recommended actions for an image, hover over one of the images list to
```

Comment:
I think a typo here around `list`.
For example:
`hover over one of the images listed to` or `hover over one of the images in the list to` or `hover over one of the images to`